staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -29,6 +29,7 @@ const (
 	DeprecatedRollbackTo           = "deprecated.deployment.rollback.to"
 	DeprecatedTemplateGeneration   = "deprecated.daemonset.template.generation"
 	StatefulSetPodNameLabel        = "statefulset.kubernetes.io/pod-name"
+	StatefulSetPodIndexLabel       = "statefulset.kubernetes.io/pod-index"
```

Comment:
The agreement is to use "statefulset.kubernetes.io/pod-index", can we please change this?

@soltysh what do you suggest we do with the existing pod name label? add a corresponding one using the key `apps.kubernetes.io/pod-name` and start the deprecation process (I bet it will be painful to deprecate since it is widely used)?

Reply:
Updated to use `apps` prefix instead of `statefulset`.